### PR TITLE
Some icons

### PR DESCRIPTION
**Pull request by zelenyhleb**
Signed-off-by: Nikifor Fedorov <nikifor.fedorov@arsysop.ru>

### CODE REVIEW

**Review by ruspl-afed — CHANGES_REQUESTED** [review]
I do not see how path in about.ini corresponds to teh real image path

**Comment by zelenyhleb** [comment]
Nice finding, @ruspl-afed. Thank you!

File: bundles/ru.arsysop.loft.rgm/about.ini [hunk 1]
```diff
@@ -29,10 +29,10 @@
 aboutText=%blurb
 
 # Property "windowImage" contains path to window icon (16x16)
-# needed for primary features only
+windowImage=images/rgm16.png
```
Comment (ruspl-afed) [inline bundles/ru.arsysop.loft.rgm/about.ini:32]: One of us is missing something important: how 
`images/rgm16.png` could lead to 
`images/icon16.png` ?

File: bundles/ru.arsysop.loft.rgm/about.ini [hunk 2]
```diff
@@ -29,10 +29,10 @@
...
 
 # Property "featureImage" contains path to feature image (32x32)
-featureImage=arsysop32.png
+featureImage=images/rgm32.png
```
Comment (ruspl-afed) [inline bundles/ru.arsysop.loft.rgm/about.ini:35]: same here

**Review by ruspl-afed — CHANGES_REQUESTED** [review]
still don't see any match between image path in about.ini and real image path

**Review by ruspl-afed — APPROVED** [review]
LGTM

**Pull request merged**